src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -172,12 +173,18 @@ private void linkToCaseWithoutRevoking(
     }
 
     private List<Claim> getAllCasesBy(User user, ImmutableMap<String, String> searchString) {
-        List<CaseDetails> validCases = searchAll(user, searchString)
-            .stream()
-            .filter(caseDetails -> !isCreatedState(caseDetails))
+        return extractClaims(searchAll(user, searchString));
+    }
+
+    private List<Claim> getAllIssuedCasesBy(User user, ImmutableMap<String, String> searchString) {
+        List<CaseDetails> issuedCases = asStream(searchAll(user, searchString))
```

Comment:
can you please give this a more clear name. Are you looking to fetch all open claims. In that case maybe rename to 
`getAllOpenCasesBy`

Reply:
It may include closed or settled cases, `getAllOpenCasesBy` will be incorrect here

Reply:
> I don't want to have access to cases that haven't reached the OPEN state yet
Please check the story it says something else..Either have the story corrected or correct the code

Reply:
The story is also fine. This method is used to fetch cases for dashboard of a defendant which may include cases in all other states i.e. closed or settled.

Reply:
If the fetch is for all type of cases. Better to name the method as `getAllCaseCreatedBy` or `getAllCaseBy`.
Issued in our context mean - Created + opened = Issued

Reply:
This what the story says **Defendant dashboards don't show cases that haven't been OPEN yet.**
I am not sure how the code and the story are both correct